packages/framework/test-client-utils/src/ConsoleLogger.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryBaseLogger, ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+
+// Basic implementation of an ITelemetry Logger. This logger will be passed into a client constructor
+// and gets hooked up to the container telemetry system.
+export class ConsoleLogger implements ITelemetryBaseLogger {
+    constructor() {}
+
+    send(event: ITelemetryBaseEvent) {
+        event.Event_Time = Date.now();
+        console.log(JSON.stringify(event));
```

Comment:
this is not a complete handling of log event. we should pipe different categories to different log levels, and appropriate handled tagged logs. if we want this as an example it should be complete

Reply:
@markfields are there any other things i missed that need to be covered when logging an event

Reply:
+1 on this. @markfields would like further guidance on how to build out this logger. Currently we have no guidance for our developers since ITelemetryBaseLogger is the interface and it only has send. @ssimic2 is also thinking on the space.

Reply:
Yeah maybe a design doc is warranted, I'm not really sure of the goal here.  I can read between the lines, but it may be better to step by and clearly articulate the need rather than trying to tweak the solution.